kayuyumwaura/countzeros/countzeros/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace countzeros
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] A = { 1, 0, 5, 6, 0, 2 };
+            int count = 0;
+
+            for (int j = 0; j < (A.Length - 1); j++)
```

Comment:
j < (A.Length -1) this means that if the size of the arr is 6 you will loop from 0 -> 4 and not 5

It should be j <= (A.Length -1) or j < (A.Length)

Reply:
Thank you for the correction. I have updated the code accordingly.